gemhardware/managers/include/gem/hw/managers/optohybrid/OptoHybridMonitor.h

```diff
@@ -4,8 +4,6 @@
 #define GEM_HW_OPTOHYBRID_OPTOHYBRIDMONITOR_H
 
 #include "gem/base/GEMMonitor.h"
-#include "gem/hw/optohybrid/exception/Exception.h"
```

Comment:
Should this and following line be replaced by the newly placed includes? Smth like `gem/hw/device/exception/Exception.h`?

Reply:
Sorry, force pushed the fixes for `gemonlinedb` and this commit is no longer reachable, however, the line you highlight is removed.
If there is an include from `gem/hw/optohybrid` then i maybe forgot to delete a file (as yes, your observation is correct), but it's compiling and locally i don't have anything that would come from that location...